### DISCUSSION TITLE
How to preset preferences for new users with dbeaver hosted on server?

### DISCUSSION THREAD

**Discussion by sublimebread** [dbeaver/dbeaver#21504] [Q&A]
Is there a way to preset the DBeaver preferences for new users while hosting DBeaver community edition on a server (Windows)?
I am trying to have the preferences set so it does not prompt to create a sample database, forces drivers to download, disables sharing information and disabling the tool tip.  Is this something that is limited to the enterprise edition? Any help with this is greatly appreciated.

**Comment by E1izabeth** [comment]
Please take a look at this guide https://github.com/dbeaver/dbeaver/wiki/Admin-Manage-Preferences

**Reply by E1izabeth** [reply]
I believe we don't have preferences for all the mentioned cases. However, you could create a feature request.

**Reply by sublimebread** [reply]
Thank you for this information @E1izabeth, Would we have to copy the preference files to each new user's dbeaver directory within their appdata folder?

**Reply by E1izabeth** [reply]
Yes

**Reply by E1izabeth** [reply]
Unless you're using our Team edittion

**Reply by sublimebread** [reply]
@E1izabeth, Is the team edition the enterprise edition? Does the enterprise edition offer an easier way to preset preferences for new users? The goal would be to host the application on a server while new users would login and use dbeaver. We would like the preferences set prior to them launching the application for the first time.

**Reply by E1izabeth** [reply]
Yes, your request really sounds like Team Edition.

You can learn more about Team Edition here:
https://dbeaver.com/dbeaver-team-edition/
https://dbeaver.com/team-edition-licenses/
